Implementation_Report_3e/index.html

```diff
@@ -663,27 +665,27 @@ <h2>4. Mastering Display Color Volume</h2>
                     </p>
                     <p>
                         PNG Third Edition adds the 
-                        <a href="https://w3c.github.io/PNG-spec/#mDCv-chunk">mDCv</a>
+                        <a href="https://w3c.github.io/PNG-spec/#mDCV-chunk">mDCV</a>
                         chunk for this purpose.
                     </p>
                     <p>
                         Implemented in <strong>Blink-based browsers</strong> (Google Chrome, Microsoft Edge, Samsung Internet, etc):
                     </p>
                     <ul>
-                        <li><a href="https://chromium-review.googlesource.com/c/chromium/src/+/4748191">The PNG specification now includes “mDCv” and “cLLi” chunks that can
+                        <li><a href="https://chromium-review.googlesource.com/c/chromium/src/+/4748191">The PNG specification now includes “mDCV” and “cLLI” chunks that can
                         encode mastering and content light level metadata. This change exposes
                         such metadata so that it may be used for tone mapping.</a></li>
                     </ul>
                     <p>Implemented in <strong>FFMPEG v2</strong> for reading and writing PNG:</p>
                     <ul>
-                        <li><a href="https://ffmpeg.org/pipermail/ffmpeg-devel/2024-February/321178.html">This adds support for cLLi and mDVc chunks in the PNG specification</a></li>
+                        <li><a href="https://ffmpeg.org/pipermail/ffmpeg-devel/2024-February/321178.html">This adds support for cLLI and mDVc chunks in the PNG specification</a></li>

```

Comment:
There is a typo here, which is why find-and-replace probably didn't pick it up.
mDVc -> mDCV

Reply:
Thanks, fixed